### PR TITLE
UNSURE loss 

### PR DESCRIPTION
**Pull request by tachella**
Adding a small modification to the SURE loss such that it can **optionally** handle unknown noise levels.

### Checks to be done before submitting your PR
- [x] `python3 -m pytest tests/` runs successfully.
- [x] `black .` runs successfully.
- [x] `make html` runs successfully (in the `docs/` directory).
- [x] Updated docstrings related to the changes (as applicable).
- [x] Added an entry to the [CHANGELOG.rst](../CHANGELOG.rst).

### CODE REVIEW

**Review by Andrewwango — CHANGES_REQUESTED** [review]
Can we add a `rng` to SURE methods inits and replace `torch.randn_like` and `torch.rand_like` with `torch.empty_like(input).normal_(generator=self.rng)` and `torch.empty_like(input).uniform_(generator=self.rng)`? (similar to `deepinv.physics.noise.NoiseModel`)

File: deepinv/training/adversarial.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ class AdversarialTrainer(Trainer):
     D: Module = None
     step_ratio_D: int = 1
 
-    def setup_train(self):
+    def setup_train(self, **kwargs):
```
[the remark this answers is not in the document]
Comment (tachella) [reply deepinv/training/adversarial.py:144]: done

File: deepinv/physics/blur.py [hunk 2]
```diff
@@ -232,7 +232,7 @@ def __init__(self, filter=None, padding="valid", device="cpu", **kwargs):
         super().__init__(**kwargs)
         self.device = device
         self.padding = padding
-        self.update_parameters(filter)
+        self.update_parameters(filter, **kwargs)
```
[the remark this answers is not in the document]
Comment (tachella) [reply deepinv/physics/blur.py]: done

File: deepinv/physics/noise.py [hunk 3]
```diff
@@ -256,7 +256,7 @@ def forward(self, x, gain=None, seed: int = None, **kwargs):
             y *= self.gain
         return y
 
-    def update_parameters(self, gain, **kwargs):
+    def update_parameters(self, gain=None, **kwargs):
```
[the remark this answers is not in the document]
Comment (tachella) [reply deepinv/physics/noise.py:259]: done

File: deepinv/training/adversarial.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Andrewwango) [reply deepinv/training/adversarial.py:144]: @tachella have you pushed this change?

**Review by Andrewwango — CHANGES_REQUESTED** [review]

**Comment by tachella** [comment]
@Andrewwango I did push that change

**Review by Andrewwango — APPROVED** [review]
